lib/adhearsion-ivr/ivr_controller.rb

```diff
@@ -119,14 +127,18 @@ def on_failure(&block)
 
     def run
       @errors = 0
-      deliver_prompt
+      deliver_prompt true
```

Comment:
Following up on CS discussion we had last week: what do you think about converting this to a hash so it reads more naturally?

```Ruby
deliver_prompt interruptible: true
```